malwarefront/src/commons/api/api.js

```diff
@@ -110,6 +110,10 @@ function getObjectRelations(id) {
     return axios.get(`/object/${id}/relations`)
 }
 
+function addObjectRelation(type, parent, child) {
+    return axios.put(`/${type}/${parent}/child/${child}`)
+}
```

Comment:
I think we should not parametrize type here and use just an 'object'.
```suggestion
function addObjectRelation(parent, child) {
    return axios.put(`/object/${parent}/child/${child}`)
}
```